src/gwt/src/org/rstudio/studio/client/workbench/views/console/shell/Shell.java

```diff
@@ -666,6 +668,13 @@ public void onError(ServerError error)
 
             input_.clear();
          }
+         else if (keyCode == KeyCodes.KEY_D &&
```

Comment:
It does, although if the user has explicitly defined their own `Ctrl + D` shortcut it will override this usage. (IIUC this is because the shortcut manager handles shortcuts in the capture phase rather than the bubble phase, whereas key handlers here get registered in the bubble phase)